lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -15,7 +15,7 @@ include_once 'dosomething_user.theme.inc';
  */
 function dosomething_user_preprocess_page(&$vars) {
   $school_api_endpoint = variable_get('dosomething_user_school_api_endpoint', 'http://lofischools.herokuapp.com/search');
-  $date_format = (dosomething_settings_get_geo_country_code() === 'US') ? 'MM/DD/YYYY' : 'DD/MM/YYYY';
```

Comment:
The client side validation is happening [here], (https://github.com/DoSomething/phoenix/blob/ecfc90e864f949bdb4ecce8bc9dc281428073e27/lib/modules/dosomething/dosomething_user/dosomething_user.module#L24), that is where we set the `dateFormat` setting. 

This code is just dealing with the placeholder text for the field, but I think I'm just confused because I was expecting it to look more like:

`$date_format = (dosomething_settings_get_geo_country_code() === 'US') ? 'MM/DD/YYYY' : t('DD/MM/YYYY')` 

Unless, what you are saying is that POEditor will see `MM/DD/YYYY` and change it accordingly?